corehq/apps/receiverwrapper/views.py

```diff
@@ -61,6 +62,8 @@
 PROFILE_LIMIT = os.getenv('COMMCARE_PROFILE_SUBMISSION_LIMIT')
 PROFILE_LIMIT = int(PROFILE_LIMIT) if PROFILE_LIMIT is not None else 1
 
+logger = logging.getLogger(__file__)
```

Comment:
Is there a reason you are using `__file__` instead of `__name__` here? I have not seen that before.

```suggestion
logger = logging.getLogger(__name__)
```

Reply:
Nope, oversight on my part. __file__ would be a mistake here, as it would cause installations to different directories to create different loggers.